domains/ethanytangcodes.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "my own website",
+  "repo": "https://github.com/is-a-dev/docs",
+  "owner": {
+    "username": "ethanytangcodes",
+    "email": "ethantytang11@gmail.com"
+  },
+  "records": {
+    "URL": "ethanytangcodes.github.io"
```

URL should contain protocol
```suggestion
    "URL": "https://ethanytangcodes.github.io"
```